### PR TITLE
added systemd service files, partitioning by default, move docs to .rst

### PR DESCRIPTION
**Pull request by jodok**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+By default the adapter will listen on port ``9268``, and talk to the local CrateDB running on port ``4200``.
+This is configurable via command line flags, which you can see by passing the ``-h`` flag.
+
+Prometheus Configuration
+========================
+
+Add the following to your `prometheus.yml`::
+
+  remote_write:
```
Comment (chaudum) [inline README.rst]: if you want correct syntax highlighting you can write

```
Add the following to your ``prometheus.yml``:

.. code-block:: yaml

  remote_write:
    - url: ...
  ...
```

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  ./crate_adapter
+
+By default the adapter will listen on port ``9268``, and talk to the local CrateDB running on port ``4200``.
+This is configurable via command line flags, which you can see by passing the ``-h`` flag.
+
+Prometheus Configuration
+========================
+
+Add the following to your `prometheus.yml`::
```
Comment (chaudum) [inline README.rst]: double backtick

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     - url: http://localhost:9268/write
+  remote_read:
+     - url: http://localhost:9268/read
+
+The adapter also exposes Prometheus metrics on ``/metrics``, and can be scraped in the usual fashion.
+
+Running with systemd
+====================
+
+Copy `<crate_adapter.service>`_ to ``/etc/systemd/system/crate_adapter.service`` and run::
```
Comment (chaudum) [inline README.rst]: you can also link the service file by running:
`sudo systemctl link $(pwd)/crate_adapter.service`
https://www.freedesktop.org/software/systemd/man/systemctl.html#link%20PATH%E2%80%A6

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
+==========================
+CrateDB Prometheus Adapter
+==========================
+
+This is an adapter that accepts Prometheus remote read/write requests,
+and sends them on to CrateDB. This allows using CrateDB as long term storage
+for Prometheus.
+
+Requires CrateDB 2.2.
```
Comment (chaudum) [inline README.rst]: Requires CrateDB 2.2.0 or greater.

**Comment by jodok** [comment]
@chaudum addressed your comments.

**Pull request merged**